src/lib/Microsoft.Health.Common/Telemetry/Metrics/Dimensions/ErrorSource.cs

```diff
@@ -0,0 +1,25 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Common.Telemetry
+{
+    public enum ErrorSource
+    {
+        /// <summary>
+        /// A service error.
+        /// </summary>
+        Service,
+
+        /// <summary>
+        /// A user error.
+        /// </summary>
+        User,
+
+        /// <summary>
+        /// An error with an undefined source.
+        /// </summary>
+        Undefined,
```

Comment:
nit: I would place this at the top since that would be taken as the default if not set.